src/SFA.DAS.EmployerIncentives.Api/Authentication/AuthenticationExtensions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerIncentives.Configuration;
+
+namespace SFA.DAS.EmployerIncentives.Api.Authentication
+{
+    public static class AuthenticationExtensions
+    {
+        public static IServiceCollection AddApiAuthentication(this IServiceCollection services, IConfiguration config)
```

Comment:
This is already defined elsewhere in the solution

Reply:
Can we discuss?

Reply:
shared project